src/popup/APIService.js

```diff
@@ -77,7 +78,7 @@ const callAPI = async function (
       console.error(err);
     }
 
-    if (err.response.status === 401) {
+    if (err.response.status === 401 && !global.isBackgroundJS) {
       handle401Error();
```

Comment:
this isn't part of the PR but I wonder should we add `await` in front of `handle401Error` here?

Reply:
fixed on 493574a0296cbab6de218ea8de6cda8dbbb09ae1